detectors/inferred-services-detectors/POST_Detector_error_rate.sh

```diff
@@ -0,0 +1,90 @@
+curl --location 'https://api.us1.signalfx.com/v2/detector' \
+--header 'Content-Type:  application/json' \
+--header 'X-SF-TOKEN: REPLACEME' \
```

Comment:
Might I suggest pulling from an environment variable in these scripts and noting the env var to use in the README so folks don't have to put their key in a file?

Thoughts?